src/sql/parts/common/customInputConverter.ts

```diff
@@ -48,6 +49,16 @@ export function convertEditorInput(input: EditorInput, options: IQueryEditorOpti
 			let queryPlanInput: QueryPlanInput = instantiationService.createInstance(QueryPlanInput, queryPlanXml, 'aaa', undefined);
 			return queryPlanInput;
 		}
+
+		//Notebook
+		uri = getNotebookEditorUri(input);
+		if(uri){
+			let notebookData: string = fs.readFileSync(uri.fsPath);
```

Comment:
>notebookData [](start = 7, length = 12)

Can you add the comment that we need to pass in notebook data either through notebook input or notebook service?

Reply:
Done